From df999982325b0d2d71ebddcde0fd36b4bf7d4863 Mon Sep 17 00:00:00 2001
From: Nathaniel Russell <46272571+nater1983@users.noreply.github.com>
Date: Sat, 13 Jan 2024 16:04:30 -0600
Subject: [PATCH 1/2] Update metadata.json

As of 46.alpha I have found that adding 46 into the shell version allows appindicator to work of Gnome 46 with out any other intervention.
---
 metadata.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index f817b0d..e690699 100644
--- a/metadata.json
+++ b/metadata.json
@@ -1,6 +1,6 @@
 {
     "shell-version": [
-        "45"
+        "45", "46"
     ],
     "gettext-domain": "AppIndicatorExtension",
     "settings-schema": "org.gnome.shell.extensions.appindicator",

From 1cbb31c50ab7e0af5e22a8b42e1c3ddcdf6d9fd1 Mon Sep 17 00:00:00 2001
From: Nathaniel Russell <46272571+nater1983@users.noreply.github.com>
Date: Mon, 12 Feb 2024 11:24:53 -0600
Subject: [PATCH 2/2] Update metadata.json

Co-authored-by: Marco Trevisan <mail@3v1n0.net>
---
 metadata.json | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index e690699..432a7bb 100644
--- a/metadata.json
+++ b/metadata.json
@@ -1,6 +1,7 @@
 {
     "shell-version": [
-        "45", "46"
+        "45",
+        "46"
     ],
     "gettext-domain": "AppIndicatorExtension",
     "settings-schema": "org.gnome.shell.extensions.appindicator",
